### ISSUE TITLE
Add to 2.1.1 understanding if keyboard interactions that don't work when AT is enabled fail or pass (was: Add to understanding of 2.1.1 a paragraph about mobile)

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag3#199]
There is a proposed SC for WCAG 2.1 that keyboard should work when AT is running.
https://github.com/w3c/wcag21/issues/62
It is intended to address mobile AT running on Talkback and an iPone. However, this is already required in WCAG 2.0 and I'm nervous that adding a new SC to 2.1 will undermine the requirements of the current 2.1.1 ans the accessibility support conformance requirement.

Therefore, I'm proposing an update to the understanding document for WCAG 2.0 SC 2.1.1. If approved I can create the pull request.

## Add to understanding
All functions available by keyboard are still available by keyboard after accessibility-supported assistive technology that remaps keystrokes is turned on. It is not always possible for users to activate arbitrary keys on a keyboard, as these may be reserved for commands to operate an assistive technology. Assistive technologies may intercept key presses before they reach the content (e.g. JavaScript on a webpage), except in very specific circumstances (such as when a text entry field receives focus).

As such, authors must not assume that users are able to successfully activate any particular key controls arbitrarily. This success criteria would apply to the assistive technology that is accessibility-supported and remaps the keyboard.

## Add to Benefits

This ensures that all users can operate the functionality when the keyboard is used with the assistive technology.

**Comment by mbgower** [comment]
> under the current wording of the SC ... yes, those would all be technically passes, but obviously horrible UX

The second and third scenarios would only be horrible UX to people who rely on or predominantly use keyboard for navigation and operation. That seems to place a massive burden on many users with disabilities, who as a group are much more likely to rely on the keyboard API. Excepting the first scenario (where _no one_ can type into an input as expected), these two "horrible UX" scenarios would seem to have almost zero effect on users who use a pointer for operation -- i.e., the majority of users without disabilities. As such, they are much more likely to be _implemented_ without being caught as a problem (and I'm assuming we can agree this IS an accessibility problem, even if you don't agree it can be mapped to something in WCAG).

I've encountered less egregious examples of keyboard mapping that still caused major problems for blind users. I have always failed those under accessibility supported:

>The way that the [web content technology](https://www.w3.org/TR/WCAG22/#dfn-technologies) is used must be supported by users' assistive technology (AT). This means that the way that the technology is used has been tested for interoperability with users' assistive technology in the [human language(s)](https://www.w3.org/TR/WCAG22/#dfn-human-language-s) of the content
